app/lib/features/news/pages/add_news_page.dart

```diff
@@ -285,17 +282,25 @@ class _AddNewsState extends ConsumerState<AddNewsPage> {
 
   Widget slideImagePostUI(BuildContext context) {
     final imageFile = selectedNewsPost!.mediaFile;
-    return Image.file(
-      File(imageFile!.path),
-      fit: BoxFit.contain,
+    return Container(
+      alignment: Alignment.center,
+      color: selectedNewsPost!.backgroundColor,
+      child: Image.file(
+        File(imageFile!.path),
+        fit: BoxFit.fitWidth,
```

Comment:
why not `contain`? What happens if the image is _larger_ now? It will run out of space on the top and bottom? I think `contain` was correct, no?

Reply:
Yes, it should be `contain`. I was testing something but my bad, forget to revert it.